aries-site/src/examples/components/controls/AnchorExample.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Anchor, Box, Text } from 'grommet';
+
+export const AnchorExample = () => {
+  return (
+    <Box direction="column" align="start" gap="medium">
+      <Box direction="row" gap="small">
+        <Text>Default anchor:</Text>
+        <Anchor color="brand" href="#" label="Anchor " />
```

Comment:
Since this is default, I don't think we need/want to show `color="brand"`

```suggestion
        <Anchor href="#" label="Anchor " />
```